client/components/videos-ui/index.jsx

```diff
@@ -3,42 +3,28 @@ import { Button, Gridicon } from '@automattic/components';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import moment from 'moment';
-import { useEffect, useState, useMemo } from 'react';
+import { useEffect, useState } from 'react';
 import Notice from 'calypso/components/notice';
-import useCourseQuery from 'calypso/data/courses/use-course-query';
-import useUpdateUserCourseProgressionMutation from 'calypso/data/courses/use-update-user-course-progression-mutation';
+import { useCourseData, useUpdateUserCourseProgressionMutation } from 'calypso/data/courses';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import VideoPlayer from './video-player';
 import './style.scss';
 
-const VideosUi = ( { HeaderBar, FooterBar, areVideosTranslated = true } ) => {
+const VideosUi = ( { courseSlug, HeaderBar, FooterBar, areVideosTranslated = true } ) => {
```

Comment:
Explicitly define the `courseSlug` prop as we might have different courses and it's easy to extend in the future.